script-test/playbackstrategies/modifiers/live/seekabletest.js

```diff
@@ -326,6 +339,22 @@ require(
 
           expect(player.resume).not.toHaveBeenCalledTimes(1);
         });
+
+        it('Should auto resume when paused after a seek', function () {
+          player.getSeekableRange.and.returnValue({start: 0});
+          player.getCurrentTime.and.returnValue(100);
+
+          startPlaybackAndPause(100, false);
+
+          player.getCurrentTime.and.returnValue(50);
```

Comment:
Surely this should happen after `playFrom`?

Reply:
It can't because playFrom kicks off dynamicWindowUtils which is checking getCurrentTime

Reply:
As discussed, this sort of change would be difficult given the state they're in, so we'll be pragmatic rather than perfectionist.